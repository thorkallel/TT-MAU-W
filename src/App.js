import React, { Fragment } from 'react';

import Crypto from './components/Crypto';

//Context API
import CriptoState from './context/cripto/criptoState';
import CotizacionState from './context/cotizacion/cotizacionState';

// Styles
import { Gstyles, HeaderStyles, FooterStyles } from './styles/gstyles';

function App() {

  return (
    <Fragment>

      {/* // General Styles */}
      <Gstyles />

      {/* // Header Styles */}
			<HeaderStyles />

      {/* // Footer Styles */}
			<FooterStyles />

      <CriptoState>
        <CotizacionState>
          <Crypto />
        </CotizacionState>
      </CriptoState>
    </Fragment>
  );
}

export default App;
