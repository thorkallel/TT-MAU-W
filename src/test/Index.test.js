import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";

import App from "../App";

describe("Pruebas en <App />", () => {
  test("debe mostrar <App /> correctamente", () => {

    const wrapper = shallow(
      <App />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
