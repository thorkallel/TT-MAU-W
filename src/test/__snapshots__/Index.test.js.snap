// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <App /> debe mostrar <App /> correctamente 1`] = `
<Fragment>
  <CriptoState>
    <CotizacionState>
      <Crypto />
    </CotizacionState>
  </CriptoState>
</Fragment>
`;
